src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.android.cs

```diff
@@ -1,14 +1,13 @@
-﻿using System.Diagnostics;
-using System.Runtime.Versioning;
+﻿using System.Runtime.Versioning;
 using Android.OS;
-using Android.Util;
 using Android.Views;
+using AndroidX.Core.View;
 using Microsoft.Maui.Platform;
 using Activity = Android.App.Activity;
 
 namespace CommunityToolkit.Maui.Core.Platform;
 
-[UnsupportedOSPlatform("Android"), SupportedOSPlatform("Android23.0")] // StatusBar is only supported on Android +23.0
+[SupportedOSPlatform("Android23.0")] // StatusBar is only supported on Android 23.0+
```

Comment:
From [here](https://learn.microsoft.com/en-us/dotnet/maui/supported-platforms), the minimum Android API supported by .NET MAUI is 23, so we can drop this attribute.
```suggestion
```

Reply:
Isn't it 21?
> Android 5.0 (API 21) or higher.

Reply:
Ops, you're right, 23 is for Blazor-mobile 🤦🏾‍♂️.